pkg/crc/machine/machine.go

```diff
@@ -94,6 +94,7 @@ func Start(startConfig StartConfig) (StartResult, error) {
 		if err != nil {
 			logging.ErrorF("Error creating host: %v", err)
 			result.Error = err.Error()
+			return *result, err
```

Comment:
To me, if there is any error then return it as soon as we get and that's we were not doing it. this looks good approach.